charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -111,20 +118,30 @@ spec:
         image: busybox
         imagePullPolicy: IfNotPresent
         env:
-          - name: JMX_USERNAME
+          - name: REAPER_JMX_USERNAME
             valueFrom:
               secretKeyRef:
                 name: {{ .Release.Name }}-reaper-secret-k8ssandra
                 key: username
-          - name: JMX_PASSWORD
+          - name: REAPER_JMX_PASSWORD
             valueFrom:
               secretKeyRef:
                 name: {{ .Release.Name }}-reaper-secret-k8ssandra
                 key: password
+          - name: SUPERUSER_JMX_USERNAME
+            valueFrom:
+              secretKeyRef:
+                name: {{ $superuserSecret }}
+                key: username
+          - name: SUPERUSER_JMX_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: {{ $superuserSecret }}
+                key: password
         args:
           - /bin/sh
           - -c
-          - echo -n "$JMX_USERNAME $JMX_PASSWORD" > /config/jmxremote.password
+          - echo "$REAPER_JMX_USERNAME $REAPER_JMX_PASSWORD" > /config/jmxremote.password; echo "$SUPERUSER_JMX_USERNAME $SUPERUSER_JMX_PASSWORD" >> /config/jmxremote.password;
```

Comment:
Should this be && ? Fail this job if one of the steps fails. Now first one can fail and user won't get an error